src/api/spec/factories/packages.rb

```diff
@@ -1,5 +1,10 @@
 FactoryGirl.define do
   factory :package do
     name Faker::Lorem.word
+    factory :package_with_file do
+      after(:create) do |package|
+        Suse::Backend.put("/source/#{package.project.name}/#{package.name}/somefile.txt", Faker::Lorem.paragraph)
```

Comment:
Can we use `PackageFile` class for this?
```ruby
PackageFile.new(name: 'somefile.txt', package_name: package.name, project_name: package.project.name).save!({}, Faker::Lorem.paragraph) # Using save! to have a exception instead of a nil in case of failure
```